Makefile

```diff
@@ -133,4 +133,17 @@ test:
 	cargo test --workspace --locked --features all-frequency-features
 
 integration-test:
-	./scripts/run_integration_tests.sh
\ No newline at end of file
+	./scripts/run_integration_tests.sh
+
+PHONY: version
+version:
+ifndef v
+	@echo "Please set the version with v=X.X.X-X"
+	@exit 1
+endif
+ifneq (,$(findstring v,  $(v)))
+	@echo "Please don't prefix with a 'v'. Use: v=X.X.X-X"
+	@exit 1
+endif
+	find ./ -type f -name 'Cargo.toml' -exec sed -i '' 's/^version = \"[0-9\.]*\"/version = \"$(v)\"/g' {} \;
+	cargo check
```

Comment:
:wheel_of_dharma:

Reply:
Is there any chance that this might match with multiline dependency version? (not that we have any now but I remember we had something like that before) ex:

https://github.com/Liberty30/polkadot-experiments/blob/main/pallets/announcement/Cargo.toml#L24